packages/core/core/src/services/document-service/entries.ts

```diff
@@ -26,7 +26,7 @@ const createEntriesService = (uid: UID.ContentType) => {
 
     const validData = await entityValidator.validateEntityCreation(contentType, data, {
       // Note: publishedAt value will always be set when DP is disabled
-      isDraft: !params?.data?.publishedAt,
+      isDraft: params.status ? params.status === 'draft' : !params?.data?.publishedAt,
```

Comment:
is there a situation where the status is draft and publishedAt is set? If that's the case, I feel we should fix that instead of adding this double check here 🤔

Reply:
publishedAt should be enough to know if we are dealing with a draft or not imo